crawler/file_handler.py

```
@@ -0,0 +1,179 @@
+def _extract_hwp(path: Path) -> tuple[str, str]:
+    """olefile로 HWP PrvText 스트림(UTF-16LE 평문) 추출."""
+    try:
+        import olefile
+
+        ole = olefile.OleFileIO(str(path))
+        if ole.exists("PrvText"):
+            raw = ole.openstream("PrvText").read()
+            return raw.decode("utf-16-le", errors="ignore").strip(), "olefile"
+        log.warning(f"HWP PrvText 스트림 없음: {path.name}")
+        return "", "olefile"
+    except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)